src/views/CardReferenceWidget.vue

```diff
@@ -0,0 +1,235 @@
+<!--
+  - @copyright Copyright (c) 2022 2022 Julien Veyssier <eneiluj@posteo.net>
+  -
+  - @author 2022 Julien Veyssier <eneiluj@posteo.net>
+  -
+  - @license GNU AGPL version 3 or any later version
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+  -->
+
+<template>
+	<div class="deck-card-reference">
+		<h3>
+			<DeckIcon :size="20" class="icon" />
+			<a v-tooltip.top="{ content: boardTooltip }"
+				:href="boardLink"
+				target="_blank"
+				class="link">
+				{{ board.title }}
+			</a>
+			<a v-tooltip.top="{ content: stackTooltip }"
+				:href="boardLink"
+				target="_blank"
+				class="link">
+				/ {{ stack.title }}
+			</a>
```

Comment:
It feels a bit weird at the moment I think.

Attached is a screenshot of:
1. Link to a card
2. Link to a board
3. A card shared into a conversation

![Bildschirmfoto vom 2022-09-12 16-33-23](https://user-images.githubusercontent.com/213943/189682147-5e1a17db-620a-48fd-b5e1-2d58ec234238.png)

I think case 1 and 3 should look similar, but the information (Title vs. Subline) is exactly the opposite?!

Reply:
Okay nevermind. you are showing much more details when a redis is configured. So for now I guess this is fine and we try to extend the old bit (card sharing) in the future.

Reply:
2 is not supported so it tries with the "link reference provider" which returns nothing. That's expected.

Ok I can make it look more like the card shared in the conversation.